### ISSUE TITLE
bug(many): systemic issue where svg images do not have appropriate role

### ISSUE THREAD

**Issue by zarend** [angular/components#27113]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

Many of our SVG images do not have an appropriate role, such as role="image". SVGs should have an appropriate role or have aria-hidden="true" instead.

This has caused some P4 issues for us. Would be easier to add aria-hidden="true" to Decorative images since that is not hard to do.

### Reproduction

Steps to reproduce:
1. 
2.

### Expected Behavior

Decorative SVG's have aria-hidden="true"

### Actual Behavior

Some of the decorative SVG's do not have aria-hidden

### Environment

- Angular:
- CDK/Material:
- Browser(s):
- Operating System (e.g. Windows, macOS, Ubuntu):
